rclcpp_action/src/server_goal_handle.cpp

```diff
@@ -135,6 +135,24 @@ ServerGoalHandleBase::try_canceling() noexcept
     return RCL_RET_OK == ret;
   }
 
-  return false;
+  return is_cancelable;
+}
+
+bool
+ServerGoalHandleBase::try_aborting() noexcept
+{
+  std::lock_guard<std::mutex> lock(rcl_handle_mutex_);
```

Comment:
include `<mutex>`

Reply:
fixed in https://github.com/ros2/rclcpp/pull/2953/commits/b1d935edb9d102ac913a278b840dabd87746c5b6